packages/console/package.json

```diff
@@ -23,7 +23,7 @@
     "@logto/language-kit": "workspace:*",
     "@logto/phrases": "workspace:*",
     "@logto/phrases-ui": "workspace:*",
-    "@logto/react": "1.0.0-beta.14",
+    "@logto/react": "1.0.0-beta.15",
```

Comment:
We need to also bump the react sdk version in demo-app and integration-tests

Reply:
updated